database-commons/src/main/java/io/cdap/plugin/db/DBRecord.java

```diff
@@ -306,7 +306,7 @@ protected void writeToDB(PreparedStatement stmt, @Nullable Schema.Field field, i
           break;
         case TIMESTAMP_MILLIS:
         case TIMESTAMP_MICROS:
-          stmt.setTimestamp(sqlIndex, Timestamp.from(record.getTimestamp(fieldName).toInstant()));
+          stmt.setTimestamp(sqlIndex, Timestamp.valueOf(record.getTimestamp(fieldName).toLocalDateTime()));
```

Comment:
Is it possible to add a unit test for this change as well please??

Reply:
Added